### PR TITLE
Kinetics 400 cpu example

### PR DESCRIPTION
**Pull request by leonardozcm**

### CODE REVIEW

File: orcaexample/kinetics.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+loss_fun = losses.get_loss_func(cfg.MODEL.LOSS_FUNC)(reduction="mean")
+
+if args.backend == "bigdl":
+    net = model_creator(cfg)
+    optimizer = optim_creator(model=net, config=cfg)
+    orca_estimator = Estimator.from_torch(model=net,
+                                          optimizer=optimizer,
+                                          loss = loss_fun,
+                                          backend=args.backend,
+                                          config=cfg)
```
Comment (leonardozcm) [inline orcaexample/kinetics.py:90]: may we extend config type to NameSpace type like CfgNode object here, intead of only dict? @hkvision 
to achieve this we change
`worker_config = self.config.copy()` to 
```
        from copy import copy
        worker_config = copy(self.config)
```

File: orcaexample/kinetics.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+#             print("Key {} with value {} is not a valid type; valid types: {}".format(
+#                 ".".join(key_list), type(cfg_node), _VALID_TYPES), )
+#         return cfg_node
+#     else:
+#         cfg_dict = dict(cfg_node)
+#         for k, v in cfg_dict.items():
+#             cfg_dict[k] = convert_to_dict(v, key_list + [k])
+#         return cfg_dict
+
+class WrappedDataLoader(DataLoader):
```
Comment (leonardozcm) [inline orcaexample/kinetics.py]: For third-party codebase, shall we add args to wrap it inside estimator?

File: .vscode/.env [hunk 3]
```diff
@@ -0,0 +1 @@
+PYTHONPATH="/home/chriskafka/project/slowfast:/home/chriskafka/project/BigDL/python/orca/src:/home/chriskafka/project/BigDL/python/orca/src/..:/home/chriskafka/project/BigDL/python/dllib/src:/home/chriskafka/develop/spark-3.1.2-bin-hadoop2.7/python/lib/py4j-0.10.9-src.zip:/home/chriskafka/develop/spark-3.1.2-bin-hadoop2.7/python/lib/pyspark.zip::/home/chriskafka/project/BigDL/python/orca/src:/home/chriskafka/develop/spark-3.1.2-bin-hadoop2.7/python/lib/py4j-0.10.9-src.zip:/home/chriskafka/develop/spark-3.1.2-bin-hadoop2.7/python/lib/pyspark.zip:/home/chriskafka/project/BigDL/scala/dllib/src/main/resources/spark-bigdl.conf"
```
Comment (hkvision) [inline .vscode/.env]: remove .vscode files?

File: slowfast/utils/misc.py [hunk 4]
```diff
@@ -412,6 +412,7 @@ def launch_job(cfg, init_method, func, daemon=False):
             daemonic processes will be created
     """
     if cfg.NUM_GPUS > 1:
+        print("GPU Number is {}".format(cfg.NUM_GPUS))
```
Comment (hkvision) [inline slowfast/utils/misc.py]: remove these debug info

File: orcaexample/kinetics.py [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+def optim_creator(model, config):
+    return optim.construct_optimizer(model, config)
+
+def loss_creator(config):
+    return losses.get_loss_func(config.MODEL.LOSS_FUNC)(reduction="mean")
+
+cfg = load_config(args, args.cfg_files)
+
+loss_fun = losses.get_loss_func(cfg.MODEL.LOSS_FUNC)(reduction="mean")
```
Comment (hkvision) [inline orcaexample/kinetics.py]: I suppose for ray and spark backends, loss can be an instance as well and so that we can remove loss_creator?

File: orcaexample/kinetics.py [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+if args.backend == "bigdl":
+    net = model_creator(cfg)
+    optimizer = optim_creator(model=net, config=cfg)
+    orca_estimator = Estimator.from_torch(model=net,
+                                          optimizer=optimizer,
+                                          loss = loss_fun,
+                                          backend=args.backend,
+                                          config=cfg)
+    orca_estimator.fit(data=train_loader_creator(cfg, 0),
```
Comment (hkvision) [inline orcaexample/kinetics.py]: 0 may not be meaningful here

File: orcaexample/kinetics.py [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+    default=None,
+    nargs=argparse.REMAINDER,
+)
+args = parser.parse_args()
+
+if args.cluster_mode == "local":
+    init_orca_context(memory="12g", cores=8)
+elif args.cluster_mode.startswith("yarn"):
+    hadoop_conf = os.environ.get("HADOOP_CONF_DIR")
+    invalidInputError(
```
Comment (jason-dai) [inline orcaexample/kinetics.py:38]: this is an internal API, not for external use. Just use `require` for user application code.

Comment (hkvision) [reply orcaexample/kinetics.py:38]: Then the check of invalidInputError should be removed from example style check? Not sure if it is easy to do this.
